RELEASE.md

```diff
@@ -106,7 +106,33 @@ Assuming the GitHub release tarball correctly installs and passes its tests, you
         pip install scikit-bio
         python -m skbio.test
 
-If this succeeds, the release appears to be a success!
+    If this succeeds, the pypi release appears to be a success.
+
+6. Next, we'll prepare and post the release to [anaconda.org](http://www.anaconda.org).
+
+    You'll need to have ``conda-build`` and ``anaconda-client`` installed to perform these steps. Both can be conda-installed. First, log into anaconda using the following command. You should log into the ``biocore`` anaconda account (if you don't have login info, get in touch with [@gregcaporaso](https://github.com/gregcaporaso) who is the owner of that account).
+
+        anaconda login
+
+    Due to its C extensions, releasing scikit-bio packages for different platforms will require you to perform the following steps on each of those platforms. For example, an ``osx-64`` package will need to be built on OS X, and a ``linux-64`` package will need to be built on 64-bit Linux. These steps will be the same on all platforms, so you should repeat them for every platform you want to release for.
```

Comment:
If only we had someone in the lab who used linux to compile the releases for us!